spec/board_spec.rb

```diff
@@ -28,13 +28,21 @@
   end
 
   it 'should have an attribute for a inverted hash of game_board' do
```

Comment:
let's refactor this test when we're done with iteration 3... "done is better than pretty"...allegedly